apps/coscrad-frontend/src/configurable-front-matter/data/configurable-front-matter.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Should this file be checked in?

Reply:
No! Good catch! Some kind of .gitignore difference between branches?

Note that this is not super sensitive, but still not meant to be checked in.

Reply:
I've explicitly removed and gitignored it on this branch too.